Portfolio project

ginabeki opened the pull request:
# My portfolio project
In this project, I added two sections:-
- The first section I added the navigation bar.
- The second section I added content to the heading section.
- Styled the page using flex layout.
- Styled the page using mobile first workflow
### Screenshot:-
Below is a snapshot of the project's progress
![Screenshot from 2022-09-21 15-33-27](https://user-images.githubusercontent.com/59937419/191510853-11cbe546-a0ef-425a-8eba-1264d8809764.png)

nedjwab commented on index.html:
```diff
@@ -0,0 +1,43 @@
+    />
+  </head>
+  <body>
+    <div>
```
[ ] kindly put the site navigation inside the` header ` tag, it's one of the HTML best practices. 🤓

nedjwab commented on index.html:
```diff
@@ -0,0 +1,43 @@
+          <img class="shape-1" src="images/Shape.png" alt="shape" />
+          <img class="shape-2" src="images/Shape-2.png" alt="shape" />
+        </div>
+        <div class="icon-menu">
```
Good job implementing the hamburger button ✔🔥

nedjwab commented on index.html:
```diff
@@ -0,0 +1,43 @@
+          <div class="rectangle-3"></div>
+        </div>
+      </nav>
+      <div class="wrap">
```
[ ] kindly use a `section `tag instead of a `div ` tag, it represents a generic standalone section of a document and it's one of the HTML best practices 🤓 .

nedjwab commented on styles.css:
```diff
@@ -0,0 +1,116 @@
+}
+
+h1 {
+  font-family: "Roboto Slab", serif;
```
[ ] kindly use the same `font-family ` used in the template which is "Playfair Display" .🤓

nedjwab commented on styles.css:
```diff
@@ -0,0 +1,116 @@
+  border: none;
+}
+
+.button:focus {
```
[ ] kindlly change the color when the button is focused 🤓🤓
your design 👇 : 
 
![pressed](https://user-images.githubusercontent.com/62156607/191519267-d30ce302-6912-459f-8768-7bbdafb03245.PNG)

The template 👀 : 

![designbuttons](https://user-images.githubusercontent.com/62156607/191519302-564d89f0-910f-43c4-a1f5-fdcd683035af.PNG)

nedjwab reviewed (changes requested):
# Changes Required ♻️
### Hi @ginabeki 

Nice job, you are almost there.

![](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.
- Linters check pass correctly ✔🔥.
- Clean design , keep it up 🔥🔥.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the required changes, but you still have some issues: 
 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

ginabeki commented:
# Changes Implemented:-
- Changed the color when the button is focused as per the design
- Changed font family to "Playfair Display"
- Replaced <div> with <section>
- Put site navigation inside the header tag

christianbiring1 reviewed (approved):
Hi @ginabeki , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
#STATUS : APPROVED :tada: 
![APPROVED](https://user-images.githubusercontent.com/97100091/191548453-044319b7-bfeb-49a7-9895-d9993c2a5181.gif)


## Highlights
:heavy_check_mark: all linters are passing
:heavy_check_mark: the design look similar to the one provided in Figma
:heavy_check_mark: the project is documented in a professional

## Optional suggestions
- [ ] Nothing to mention

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
